tracer/test/Datadog.Trace.TestHelpers/RemoteConfigTestHelper.cs

```diff
@@ -23,6 +23,16 @@ public static void SetupRcm(this MockTracerAgent agent, ITestOutputHelper output
             output.WriteLine("Using RCM response: " + response);
         }
 
+        internal static async Task<GetRcmRequest> SetupRcmAndWait(this MockTracerAgent agent, ITestOutputHelper output, IEnumerable<(object Config, string Id)> configurations, string productName)
+        {
+            var response = BuildRcmResponse(configurations, productName);
+            agent.RcmResponse = response;
+            output.WriteLine("Using RCM response: " + response);
+            var res = await agent.WaitRcmRequestAndReturnLast();
+            await Task.Delay(1000);
```

Comment:
we shouldnt need that with the `LogEntryWatcher`, but this can be for a subsequent PR as messages need some fixing